server/scripts/dbhelper.js

```diff
@@ -9,24 +9,27 @@ async function main() {
   const command = process.argv[2]
   if (command === 'create') {
     const salt = generateSalt()
+    const secretKey = generateSecretKey()
     const user = new Users({
-      username: process.argv[3],
-      hashedApiKey: await hashPass(process.argv[4], salt),
-      email: process.argv[5],
+      email: process.argv[3],
+      accessKey: generateAccessKey(),
+      hashedSecretKey: await hashPass(secretKey, salt),
       salt: salt
     })
     console.log(await user.save())
+    console.log('SECRET KEY:', secretKey)
```

Comment:
log access key too?

Reply:
maybe with the email

Reply:
It's all logged in the `console.log(await user.save())`. The only thing not logged in the `user.save()` is the private key, because that never hits the DB.